taskcluster/ci/config.yml

```diff
@@ -196,3 +185,40 @@ release-promotion:
                     - "hectorz+mozillaonline-priv-webext-promote@mozilla.com"
                     - "mkaply+mozillaonline-priv-webext-promote@mozilla.com"
                     - "awagner+mozillaonline-priv-webext-promote@mozilla.com"
+        # configure normandy-privileged webextension email addresses for
+        # notifications
+        normandy-privileged:
+            by-phase:
+                build:
+                    - "aki+normandy-priv-webext-build@mozilla.com"
+                    - "rdalal+normandy-priv-webext-build@mozilla.com"
+                    - "mcooper+normandy-priv-webext-build@mozilla.com"
+                promote:
+                    - "aki+normandy-priv-webext-promote@mozilla.com"
+                    - "rdalal+normandy-priv-webext-promote@mozilla.com"
+                    - "mcooper+normandy-priv-webext-promote@mozilla.com"
+        # configure system addons email addresses for notifications
+        system:
+            by-phase:
+                build:
+                    - "aki+system-addons-build@mozilla.com"
+                    - "rdalal+system-addons-build@mozilla.com"
+                    - "mcooper+system-addons-build@mozilla.com"
+                promote:
+                    - "aki+system-addons-promote@mozilla.com"
+                    - "rdalal+system-addons-promote@mozilla.com"
+                    - "mcooper+system-addons-promote@mozilla.com"
+        # Deprecated 'privileged' add-ons email addresses for notifications
```

Comment:
Let's remove these. xpi-manifest doesn't look at the source repos' `.taskcluster.yml`; only xpi-template does.

Reply:
2bebb23